keps/sig-cluster-lifecycle/0035-20190128-addons-via-operators.md

```diff
@@ -0,0 +1,181 @@
+KEP: Addons via Operators
```

Comment:
All content has to follow the YAML front matter. See https://jekyllrb.com/docs/front-matter/